MatrixSDK/JSONModels/AutoDiscovery/MXWellKnownAuthentication.h

```diff
@@ -33,6 +33,8 @@ NS_ASSUME_NONNULL_BEGIN
 @property (nonatomic, readonly) NSString *issuer;
 @property (nonatomic, readonly, nullable) NSString *account;
 
+-(NSURL * _Nullable) getMasLogoutDeviceURLFromDeviceID: (NSString * ) deviceID;
```

Comment:
Totally minor nit: we're getting the url for a device ID. Also, I'm not 100% sure MAS is strictly always the OIDC provider.
```suggestion
-(NSURL * _Nullable) getLogoutDeviceURLForDeviceID: (NSString * ) deviceID;
```